stdlib/inspect.pyi

```diff
@@ -247,19 +247,22 @@ class BoundArguments:
 def getclasstree(classes: list[type], unique: bool = ...) -> list[Any]: ...
 def walktree(classes: list[type], children: dict[Type[Any], list[type]], parent: Type[Any] | None) -> list[Any]: ...
 
-class ArgSpec(NamedTuple):
-    args: list[str]
-    varargs: str | None
-    keywords: str | None
-    defaults: Tuple[Any, ...]
+if sys.version_info < (3, 11):
+    class ArgSpec(NamedTuple):
+        args: list[str]
+        varargs: str | None
+        keywords: str | None
+        defaults: Tuple[Any, ...]
 
 class Arguments(NamedTuple):
     args: list[str]
     varargs: str | None
     varkw: str | None
 
 def getargs(co: CodeType) -> Arguments: ...
-def getargspec(func: object) -> ArgSpec: ...
+
+if sys.version_info < (3, 11):
+    def getargspec(func: object) -> ArgSpec: ...
```

Comment:
Nit: It might be cleaner to put `class ArgSpec` and `getargspec()` under the same `if` statement, so it is clear that they are related.

Reply:
I only did it like that to make the diff more readable; I'll make the change 🙂